x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -544,6 +544,19 @@ export const getCatchupTuples = ({
   return catchupTuples;
 };
 
+/**
+ * Takes the rule schedule fields `interval` and `lookback` and uses them to calculate the `from` value for a rule
+ *
+ * @param interval moment.Duration representing the interval on which the rule runs
+ * @param lookback moment.Moment representing the rule's additional lookback
+ * @returns moment.Moment representing the rule's 'from' property
+ */
+export const calculateFromValueWithRuleScheduleFields = (interval: string, lookback: string) => {
+  const parsedInterval = parseInterval(interval) ?? moment.duration(0);
+  const parsedFrom = parseInterval(lookback) ?? moment.duration(0);
+  return parsedFrom.asSeconds() + parsedInterval.asSeconds();
```

Comment:
Let's make it return fully formatted from value:

```suggestion
  const duration = parsedFrom.asSeconds() + parsedInterval.asSeconds();
  return `now-${duration}s`
```